new-evaluation.css

```diff
@@ -0,0 +1,143 @@
+		html {
+			height: 100%;
+			box-sizing: border-box;
+		}
+
+		body {
+			min-height: 100%;
+			position: relative;
+		
+		}
+		
+		.header {
+			height: 70px;
+			border-bottom: 1px solid grey;
+		}
+
+		#logo {
+			width: auto;
+			height: 50px;
+			line-height: 70px;
+			vertical-align: middle;
+		}
+
+		.div_form {
+			padding-top: 55px;
+			margin: 0 auto ;
+			width: 1000px;
+			height: 40px;
+		}
+
+		.input {
```

Comment:
Lets not couple the class names with tags names to tightly. I'm sure you can come up with a better name to describe this class.